### PR TITLE
`Development`: Add support for HTTP/3

### PR DESCRIPTION
**Pull request by bensofficial**

### CODE REVIEW

File: roles/firewall/templates/rules_proxy.v4.j2 [hunk 1]
```diff
@@ -4,7 +4,10 @@
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
-:OUTPUT ACCEPT [0:0]
+:OUTPUT ACCEPT [8:908]
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
```
Comment (kliwniloc) [inline roles/firewall/templates/rules_proxy.v4.j2:10]: Why are the rules duplicated here?

File: roles/firewall/templates/rules_proxy.v6.j2 [hunk 2]
```diff
@@ -4,19 +4,22 @@
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
-:OUTPUT ACCEPT [0:0]
+:OUTPUT ACCEPT [54:7261]
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
+-A INPUT -p udp -m udp --dport 443 -j ACCEPT
```
Comment (kliwniloc) [inline roles/firewall/templates/rules_proxy.v6.j2:10]: Why are the rules duplicated here?

File: roles/firewall/templates/rules_proxy.v4.j2 [hunk 3]
```diff
@@ -14,19 +17,19 @@
...
 # Generated by iptables-save v1.6.1 on Thu Jun 20 10:00:00 2020
 *nat
-:PREROUTING ACCEPT [0:0]
-:INPUT ACCEPT [0:0]
-:OUTPUT ACCEPT [0:0]
-:POSTROUTING ACCEPT [0:0]
+:PREROUTING ACCEPT [797635:54478213]
+:INPUT ACCEPT [776639:47280780]
+:OUTPUT ACCEPT [7310:564961]
+:POSTROUTING ACCEPT [7310:564961]
```
Comment (kliwniloc) [inline roles/firewall/templates/rules_proxy.v4.j2:33]: This looks like an accidental staging of changes. If not: why is this here?

**Review by kliwniloc — DISMISSED** [review]
IIRC, the Nginx module supporting HTTP/3 remains in an experimental phase and requires some further tweaking in Nginx configuration parameters. Has this been sorted out in the primary branch yet, or is it still an issue?

**Review by Mtze — APPROVED** [review]
